.gitignore

```
@@ -143,4 +143,7 @@ checklink/cookies.txt
 # wandb files
 nbs/wandb/
 examples/notebooks/wandb/
-wandb/
\ No newline at end of file
+wandb/
+
+# chat files
+trl/commands/
```

should only be the scripts folder.

```suggestion
trl/commands/scripts/
```

---

Fixed in https://github.com/huggingface/trl/pull/1486/commits/eb086568001a1f11ecff4fc1c86da19dbe22271a